### PR TITLE
Fix role arn operation

### LINKED ISSUE #51: AWS assume role does not work

### PR DESCRIPTION
**Pull request by jmcshane**
I tested this in a merged branch between my two PRs, but it did seem to do the trick

Resolves #51

### CODE REVIEW

**Review by robotoer — APPROVED** [review]
👍

File: pkg/aws/aws_test.go [hunk 1]
```diff
@@ -60,7 +66,7 @@ func TestGenerateEnvironment(t *testing.T) {
 
 	for _, tc := range tests {
 		t.Run(tc.name, func(t *testing.T) {
-			plugin := aws.New(tc.region, &mockSTSAPI{expected: tc.credentials}, tc.sessionDuration)
+			plugin := aws.New(tc.region, &mockSTSAPI{expected: tc.credentials}, tc.sessionDuration, "")
```
Comment (tammersaleh) [inline pkg/aws/aws_test.go:69]: I struggle parsing Go, but is this really enough testing for this feature?  I feel like there's at least one big conditional on `if roleARN != ""` (not to mention all of the `err` checks), which makes me feel like there should be at least another test showing that both branches behave correctly, no?

Comment (jmcshane) [reply pkg/aws/aws_test.go:69]: Yeah, I need to do that better. Given the current test coverage state, I was going to put that together in a separate PR to get some broad coverage

Comment (jmcshane) [reply pkg/aws/aws_test.go:69]: See #57

**Pull request merged**